src/applications/personalization/dashboard/helpers.jsx

```diff
@@ -84,6 +84,23 @@ export const formTitles = Object.keys(formBenefits).reduce((titles, key) => {
   return titles;
 }, {});
 
+export const formDescriptions = Object.keys(formBenefits).reduce(
+  (descriptions, key) => {
+    let formNumber;
+    if (key === '40-10007' || key === 'VIC') {
+      formNumber = '';
+    } else if (key === '1010ez') {
+      formNumber = '(10-10EZ)';
+    } else {
+      formNumber = `(${key})`;
+    }
+    const formDescription = `${formBenefits[key]} application ${formNumber}`;
```

Comment:
nit: I realize this code was just moved here from another file, but it would be nice if the trailing whitespace in the `formDescription` string was removed in the case of 40-10007 and VIC.

Reply:
Good catch/question, JSX won't display the trailing whitespace.
![image](https://user-images.githubusercontent.com/16051172/47375296-cdc4af80-d6a4-11e8-9d98-d1ab98d0eea3.png)
Here are some resources on JSX and whitespace: [String Literals](https://reactjs.org/docs/jsx-in-depth.html#string-literals-1), [JSX Whitespace](https://reactjs.org/blog/2014/02/20/react-v0.9.html#jsx-whitespace).